pkgs/by-name/pr/prometheus-cpp/package.nix

```
@@ -1,46 +1,54 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, cmake
-, gbenchmark
-, gtest
-, civetweb
-, zlib
-, curl
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  civetweb,
+  curl,
+  gbenchmark,
+  gtest,
+  zlib,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "prometheus-cpp";
-  version = "1.1.0";
+  version = "1.3.0";
 
   src = fetchFromGitHub {
     owner = "jupp0r";
-    repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-qx6oBxd0YrUyFq+7ArnKBqOwrl5X8RS9nErhRDUJ7+8=";
+    repo = "prometheus-cpp";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-XQ8N+affKVqn/hrMHWg0eN+0Op6m9ZdVNNAW0GpDAng=";
```

```suggestion
    tag = "v${finalAttrs.version}";
    hash = "sha256-XQ8N+affKVqn/hrMHWg0eN+0Op6m9ZdVNNAW0GpDAng=";
```
Would be nice to use these instead